maps/snaxi.dm

```diff
@@ -41,9 +41,56 @@
 
 /datum/map/active/New()
 	.=..()
+	
 	research_shuttle.name = "Research and Mining Shuttle"
 	research_shuttle.req_access = list() //It's shared by miners and researchers, so remove access requirements
 
+// Making nodes every 5*5 tiles in a 100*100 radius
+// This makes (100*100)/(10*10) = 100 nodes
```

Comment:
It'd be better to make this a define, either as a variable that gets unpacked or as the equation in lambda notiation at the beginning of the file, so it isn't some magic number pulled outta nowhere. Much easier to debug and adjust. 100 could be too many, too little- or the number itself could be randomized.
And is snaxi.dm _still_ called "Datum Boxstation" up top? Jesus christ, how lazy can you get?

Reply:
Well that last part wasn't me

Reply:
I know. Someone should make a pull request to change that _just_ to rub it in.